src/App.tsx

```diff
@@ -15,71 +15,105 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  handlerSelect = (good: string) => {
+    return () => {
```

Comment:
For what are you using return

Reply:
I believe Bogdan wanted to emphasise that there is no need to return something from that function, you can have it like that: 
```
  handlerSelect = (good: string) => {
      this.setState({ selectedGood: good });
  };
  ```

Reply:
still not fixed